lib/crypto/src/crypto.erl

```diff
@@ -1411,7 +1411,7 @@ verify(Algorithm, Type, Data, Signature, Key) ->
                         Result :: boolean().
 
 verify(Algorithm0, Type0, Data, Signature, Key, Options) ->
-    {Algorithm, Type} = sign_verify_compatibility(Algorithm0, Type0, Data),
+    {Algorithm, Type} = handled_digest_type(Algorithm0, Type0, Data),
```

Comment:
I find the function name confusing, what does the function do? Can it have a better name describing what it does?

Reply:
The function handles the documented digest type and converts it to what crypto nif internally uses.
The argument is called DigestType in the spec and documentation. Before a conversion was only needed for backwards compatibility handling of the DSA algorithm and was symmetrical for sign and verify. For some reason it is asymmetrical what is needed for sign and verify for the EDDSA algorithm.  This however is internal to the crypto application and could perhaps be part of a future enhancement of crypto. In the API we use the already existing type of none as EDDSA uses the identity function as digest here (in math terms return the input unchanged) .

Reply:
I still think that `handled_digest_type` feels like a spelling error or a bad name.
I think it is transforming or converting or normalizing a digest type. If you like to keep handle then I propose `handle_digest_type` not **`handled`**.

Reply:
You are right the last d was a typo, but lets call it internal_verify_alg_types instead! Hope that feels clearer to you.

Reply:
ok